unicode-org/icu
# ICU-22535 CLDR 45 beta3 to main

##### Checklist

- [ ] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-22535 
- [ ] Required: The PR title must be prefixed with a JIRA Issue number. 
- [ ] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [ ] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [ ] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

Beta 3 integration that covers minor CLDR updates since beta2.

- Check for pairing bidi markup chars (illegal); remove 2 (unpaired) RLO in ff_Adlm [CLDR-16034](https://unicode-org.atlassian.net/browse/CLDR-16034)
- Currency change for CW,SX fixed to be in 2025, not 2024
- Update SPDX where it was missing [CLDR-17110](https://unicode-org.atlassian.net/browse/CLDR-17110)

ALLOW_MANY_COMMITS=true


[CLDR-16034]: https://unicode-org.atlassian.net/browse/CLDR-16034?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
